pkg/utils/informer/informermanager.go

```diff
@@ -101,32 +114,36 @@ type informerManagerImpl struct {
 	ctx    context.Context
 	cancel context.CancelFunc
 
-	// informerFactory is the client-go built-in informer factory that can create an informer given a gvr.
-	informerFactory dynamicinformer.DynamicSharedInformerFactory
-
+	cache  Cache.Cache
+	scheme *runtime.Scheme
 	// the apiResources map collects all the api resources we watch
 	apiResources  map[schema.GroupVersionKind]*APIResourceMeta
 	resourcesLock sync.RWMutex
 }
 
 func (s *informerManagerImpl) AddDynamicResources(dynResources []APIResourceMeta, handler cache.ResourceEventHandler, listComplete bool) {
 	newGVKs := make(map[schema.GroupVersionKind]bool, len(dynResources))
-
 	addInformerFunc := func(newRes APIResourceMeta) {
-		dynRes, exist := s.apiResources[newRes.GroupVersionKind]
+		_, exist := s.apiResources[newRes.GroupVersionKind]
 		if !exist {
-			newRes.isPresent = true
+			// TODO: how to add GVK to scheme?
```

Comment:
The old method (using dynamic informer factory) automatically added the dynamic resources to the scheme, but informer cache does not. Currently, running into an issue where the not how to add the dynamic resources to the scheme using only the GVK?